sdk/ml/azure-ai-ml/azure/ai/ml/_schema/pipeline/control_flow_job.py

```diff
@@ -133,3 +133,15 @@ def _binding_handler(obj):
             # use str to serialize input/output builder
             result._items = json.dumps(result.items, default=_binding_handler)
         return result
+
+
+class FLScatterGatherSchema(ControlFlowSchema):
```

Comment:
Not sure if we really need this inheritance

Reply:
Not that I'm aware of other than mental bookkeeping